lib/models/result.js

```diff
@@ -0,0 +1,38 @@
+"use strict" ;
+
+module.exports = class Result {
+  /* This result model is based on the QualityReportResult class found in 
+    https://github.com/projectcypress/quality-measure-engine/blob/master/lib/qme/quality_report.rb
+    The model is designed to handle both Cat1 and Cat3 aggregate results
+    */
+
+  measure_id = '';
+  sub_id = '';
+  test_id = '';
+  effective_date = 0;
+  filters = {};
+  prefilter = {};
+  calculation_time = Date.now();
+  status = {"state": "unknown", "log": []};
+
+  population_ids = {};
+  IPP = 0;
```

Comment:
Does it make sense to define these in a more configurable location? Also, I think you might have missed a population. For Bonnie (and related HDS, Bundler, etc code), we look here for these: https://github.com/projectcypress/health-data-standards/blob/8143617faac34658c134ddeb3115c8f80dd1e6ec/lib/hqmf-model/population_criteria.rb

Reply:
Cypress only uses those I think, but if Bonnie uses more than I will definitely add those.

Reply:
Just added the missing population. Regarding configurable location, you might be right, but in the past at least, Cypress has defined them in the report object as (I guess) the configurable location, since it is the primary location where we are keeping track of them. At least for now, it makes sense to have it there as a preliminary pass.